Sources/Profile-View.php

```diff
@@ -623,11 +623,10 @@ function showPosts($memID)
 	if ($context['is_topics'])
 		$request = $smcFunc['db_query']('', '
 			SELECT COUNT(*)
-			FROM {db_prefix}topics AS t' . ($user_info['query_see_board'] == '1=1' ? '' : '
-				INNER JOIN {db_prefix}boards AS b ON (b.id_board = t.id_board AND {query_see_board})') . '
-			WHERE t.id_member_started = {int:current_member}' . (!empty($board) ? '
-				AND t.id_board = {int:board}' : '') . (!$modSettings['postmod_active'] || $context['user']['is_owner'] ? '' : '
-				AND t.approved = {int:is_approved}'),
+			FROM {db_prefix}topics AS b' . '
```

Comment:
Once `query_see_topic_board` is added, the query here can go back to using `{db_prefix}topics AS t` along with `{query_see_topic_board}`